### DISCUSSION TITLE
Error: File not found: '/home/runner/work/_actions/suzuki-shunsuke/tfaction/main/get-target-config/dist/index.js'

### DISCUSSION THREAD

**Discussion by NimeshaPremaraja** [suzuki-shunsuke/tfaction#1961] [Deprecated (Support Request)] [answered]
### tfaction version

Version: suzuki-shunsuke/tfaction/test@a2ed2826824c787c17b28f91b85f701f793eaef8

### Overview

I'm getting following error when Running ```suzuki-shunsuke/tfaction/test``` in my GitHub Action

```Error: File not found: '/home/runner/work/_actions/suzuki-shunsuke/tfaction/main/get-target-config/dist/index.js```

### How to reproduce

tfaction-root.yaml

```yaml

```

tfaction.yaml

```yaml

```

GitHub Actions Workflow

```yaml

```

Other related code such as local Registry

```

```

GitHub Actions' log

```
```

### Expected behaviour

Should get the message as ```Success! The configuration is valid.```

### Actual behaviour

GitHub Action fails with error
```
Run suzuki-shunsuke/tfaction/test@a2ed2826824c787c17b28f91b85f701f793eaef8
Run suzuki-shunsuke/tfaction/get-target-config@main
Error: File not found: '/home/runner/work/_actions/suzuki-shunsuke/tfaction/main/get-target-config/dist/index.js'
```

**Comment by suzuki-shunsuke** [comment, marked as answer]
As of [v1.10.0](https://github.com/suzuki-shunsuke/tfaction/releases/tag/v1.10.0), we stop building JavaScript in the main branch and feature branches.

- https://github.com/suzuki-shunsuke/tfaction/pull/1913

Please use release versions.
